main.cpp

```diff
@@ -1,157 +1,166 @@
+        for example: 
+            noun: lightBulb.
+            action: turns on.  
+                This is a bad action because someone else turns on the light.  the light doesn't turn itself on. 
+            a better action would be: 
+                illuminates room
+                This is a better action, because this is what lightbulbs do.  
+            other actions that would work:
+                action: consume electricity
+                action: burn out and destroy filament.
+
+        
+ 3) write out how you'd call that action in pseudo code, in the space after the plain-english action
+ 
+ 4) If the action requires multiple words, use camelCaseToNameIt
+     don't forget the semi-colon after each statement
+ */
+
+
+//  example)
+//  Noun:       arm                     // 1)
+//  action 1:   the arm extends         // 2)
+arm.extend();                           // 3)
+//  action 2:   the arm flexes          // 2)
+arm.flex();                             // 3)
+//  action 3:   the arm rotates conter-clockwise
+arm.rotateCounterClockwise();           // 4) demonstrates camelCase
+
+
+//  1)
+//  Noun: Smartphone
+//  action 1: the smartphone makes a call
+smartphone.makeCall();
+//  action 2: the smartphone takes a picture
+smartphone.takePicture();
+//  action 3: the smartphone plays a tune
+smartphone.playTune();
+    
+//  2)
+//  Noun: Watch
+//  action 1: tells the time
+watch.tellTime();
+//  action 2: adjusts the minutes
+watch.adjustMinutes();
+//  action 3: sets the alarm
+watch.setAlarm();
+    
+//  3)
+//  Noun: Vacuum cleaner
+//  action 1: the vacuum cleaner starts cleaning
+vacuumCleaner.start();
+//  action 2: the vacuum cleaner stops cleaning
+vacuumCleaner.stop();
+//  action 3: the vacuum cleaner connects to charger
+vacuumCleaner.recharge();
+    
+//  4)
+//  Noun: Hand
+//  action 1: the hand waves
+hand.wave();
+//  action 2: the hand grabs an object
+hand.grabObject();
+//  action 3: the hand shakes another hand
+hand.shakeAnotherHand();
+
+//  5)
+//  Noun: Eyes
+//  action 1: the eyes read a text
+eyes.readText();
+//  action 2: the eyes blink
+eyes.blink();
+//  action 3: the eyes look to the left
+eyes.lookLeft();
+
+//  6)
+//  Noun: Dog
+//  action 1: a dog barks
+dog.bark();
+//  action 2: a dog eats
+dog.eat();
+//  action 3: a dog sleeps
+dog.sleeps();
+    
+//  7)
+//  Noun: Fan
+//  action 1: the fan rotates clockwise
+fan.rotateClockwise();
+//  action 2: the fan rotates counter-clockwise
+fan.rotateCounterClockwise();
+//  action 3: the fan stops rotating
+fan.stop();
+
+//  8)
+//  Noun: Kite
+//  action 1: the kite moves up
+kite.moveUp();
+//  action 2: the kite moves down
+kite.moveDown();
+//  action 3: the kite spins
+kite.spin();
+
+//  9)
+//  Noun: Printer
+//  action 1: the printer prints in color
+printer.printsInColor();
+//  action 2: the printer prints in black and white
+printer.printsInBW();
+//  action 3: the printer needs a new cartridge
+printer.needsNewCartridge();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.